### ISSUE TITLE
Request: Code Completion ;)

### ISSUE THREAD

**Issue by pheuter** [nixeagle/ooc-mode#11]
That would be epic, wouldn't it?

**Comment by nixeagle** [comment]
Yes it would, this will take several days to get a start on implementing. I have begun work on the lexer portion of the semantic requirements. Thanks for pointing out http://github.com/nddrylliog/nagaqueen/blob/master/grammar/nagaqueen.leg#L340 to me :)

**Comment by nixeagle** [comment]
Just an update that work is happening on this over on the semantic branch :)
